### ISSUE TITLE
PushClient Module

### ISSUE THREAD

**Issue by somefakeuser** [arleyandrada/PushClient#81]
Hi,
I just purchased the module today to use it in ios+android.

I already configured the certificate for APN service and got API key and id for GCM. But, then I'm clueless how to use in my app.

Sorry, I'm very new to Appcelerator. If you could give advise  or the steps how to use it in my app and where to send the notification (ACS or APN or GCM) would be really helpful.

Thanks in advance for your assistance. 

Many Thanks & Regards,

**Comment by arleyandrada** [comment]
Thanks for reaching out!
Unfortunately we've suspended the project for now and it's been a few months since the last update.
Your module was self-descriptive and I didn't have any trouble on my side.

The major reason I chose yours was that one of your examples used parse.com with android,
which I was desperate about and didn't have much luck finding their messaging ID.
and bang! your example had the ID and everything in place ha ha.

Nao
